charts/newrelic-infrastructure-beta/values.yaml

```
@@ -0,0 +1,398 @@
+# Default values for newrelic-infrastructure.
+nameOverride: ""
+fullnameOverride: ""
+
+enableLinux: true
+enableWindows: false
+windowsOsList:
+  # - version: 1809
+  #   imageTag: 2.2.0-windows-1809-alpha
+  #   buildNumber: 10.0.17763
+  # - version: 1909
+  #   imageTag: 2.2.0-windows-1909-alpha
+  #   buildNumber: 10.0.18363
+  - version: 2004
+    imageTag: 2.2.0-windows-2004-alpha
+    buildNumber: 10.0.19041
+
+images:
+  pullSecrets: []
+  forwarder:
+    pullPolicy: IfNotPresent
+    tag: 1.20.5
+    repository: newrelic/k8s-events-forwarder
+  agent:
+    pullPolicy: IfNotPresent
+    tag: 2.7.6
+    repository: newrelic/infrastructure-bundle
+  scraper:
+    pullPolicy: IfNotPresent
+    tag: 0.1.2
+    repository: newrelic/nri-kubernetes
+
+controlPlane:
+  enabled: false
+  annotations: {}
+  tolerations:
+    - operator: "Exists"
+      effect: "NoSchedule"
+    - operator: "Exists"
+      effect: "NoExecute"
+  nodeSelector: {}
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: node-role.kubernetes.io/control-plane
+                operator: Exists
+          - matchExpressions:
+              - key: node-role.kubernetes.io/controlplane
+                operator: Exists
+          - matchExpressions:
+              - key: node-role.kubernetes.io/etcd
+                operator: Exists
+          - matchExpressions:
+              - key: node-role.kubernetes.io/master
+                operator: Exists
+  kind: DaemonSet
+  extraEnv: []
+  extraEnvFrom: []
+  extraVolumes: []
+  extraVolumeMounts: []
+  resources:
+    limits:
+      memory: 300M
+    requests:
+      cpu: 100m
+      memory: 150M
+  scraperConfig:
```

I'd call this simply
```suggestion
  config:
```